<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>op_interface</class>
 <widget class="QMainWindow" name="op_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> 操作员界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="locate">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>我的信息</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>操作员信息：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>编号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>所属仓库：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="op_code">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="op_name">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="op_stor_code">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="cur_pwd">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="new_pwd">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>300</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="con_pwd">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>330</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>当前密码：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>修改密码：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>确认密码：</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>379</x>
        <y>0</y>
        <width>31</width>
        <height>411</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="op_passwd_change">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>360</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>密码修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="op_info_change">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>信息修改</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>密码修改：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stor_name">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>150</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>100</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>仓库编号：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stor_alter">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>290</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>250</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>联系方式：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phone_num">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>250</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>150</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>仓库名称：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stor_code">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>100</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>查看仓库信息：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>200</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>仓库位置：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="locate_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>200</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>商品信息</string>
     </attribute>
     <widget class="QComboBox" name="goods_code">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>70</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>查看商品信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>70</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>商品编号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>120</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>商品名称：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>170</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>商品类型：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="goods_name">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>120</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="goods_type">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>170</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="num_units">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>220</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>220</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>数量单位：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>厂商名称：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="factory">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>270</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="goods_add">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>320</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>增加</string>
      </property>
     </widget>
     <widget class="QPushButton" name="goods_delete">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>320</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="goods_alter">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>320</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QTableView" name="info_goods">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>521</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>出入信息</string>
     </attribute>
     <widget class="QPushButton" name="record_add">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>320</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>增加</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>170</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>处理操作：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>查看出入信息</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="date">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>270</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="op_goods_code">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>70</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="op_amount">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>220</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>处理日期：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>220</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>处理数量：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>70</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>商品编号：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="op_type">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>170</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>入库</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>出库</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="record_delete">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>320</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="record_alter">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>320</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QTableView" name="info_record">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>521</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="op_num">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>120</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_84">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>120</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>处理编号：</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>统计查询</string>
     </attribute>
     <widget class="QTableView" name="info_stat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>521</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>报表信息</string>
      </property>
     </widget>
     <widget class="QComboBox" name="stat_goods_code">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>70</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="stat_num_units">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>270</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="stat_goods_name">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>120</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_79">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>商品单位：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_80">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>170</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>商品类型：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_81">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>220</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>商品数量：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_82">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>70</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>商品编号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stat_goods_amount">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>220</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_83">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>120</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>商品名称：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stat_goods_type">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>170</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="select">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>320</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
